models/staging/stg_workday.yml

```diff
@@ -22,8 +22,8 @@ models:
       - name: critical_job
         description: '{{ doc("critical_job") }}'
 
-      - name: job_profile_description
-        description: '{{ doc("job_profile_description") }}'
+      - name: job_profile_description 
```

Comment:
There are a few fields that were renamed in the staging models and need to be renamed in the yml. Please see the highlighted fields within the stg_workday.yml for all the ones I noticed needed to be updated.

Additionally, please note that any other fields that are renamed in the staging models (one of my previous comments) will also need to see the docs updated to reflect the new name.

Reply:
Renamed.